charts/openfaas-cloud/templates/ingress/ingress-auth.yml

```diff
@@ -0,0 +1,26 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: openfaas-auth-ingress
+  namespace: {{ .Values.global.coreNamespace }}
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+    nginx.ingress.kubernetes.io/limit-connections: "20"
```

Comment:
These limits would be good to template since it may not be high enough for corporate users like DX cc @csakshaug